plugins/palo_alto_pan_os/help.md

```diff
@@ -1069,8 +1142,33 @@ For the URL, include `https://` e.g. `https://10.0.0.1` or `https://myfirewall`.
 
 When using the Add External Dynamic List action, a day and time must be chosen even if they are not used.
 
+Action connection type
+
+   | Action name                       | Connection type |
+   | -----------                       | -------------------- |
+   | Add Address Object to Group       | Direct firewall |
+   | Add External Dynamic List         | Direct firewall |
+   | Add to Policy                     | Direct firewall |
+   | Check if Address in Group         | Direct firewall |
+   | Commit                            | Direct firewall |
+   | Create Address Object             | Direct firewall |
+   | Delete                            | Panorama |
+   | Edit                              | Panorama |
+   | Get                               | Panorama |
+   | Get Addresses from Group          | Direct firewall |
+   | Get Policy                        | Direct firewall |
+   | Op                                | Direct firewall |
+   | Remove Address Object from Group  | Direct firewall |
+   | Remove from Policy                | Direct firewall |
+   | Retrieve Logs                     | Direct firewall |
+   | Set                               | Panorama |
+   | Set Security Policy Rule          | Direct firewall |
+   | Show                              | Panorama |
+
+
 # Version History
 
+* 6.1.4 - Add information to every action is it work with Panorama or direct firewall connection
```

Comment:
This is not needed.

Reply:
We leave version, because we added information about connection to description of every action. If you would like we can of course reuse previous version.

Reply:
This is not needed because there is no new plugin version 6.1.4.